irohad/ordering/impl/single_peer_ordering_service.cpp

```diff
@@ -74,8 +75,11 @@ namespace iroha {
           batch_prop_mutex_);
 
       current_size_.fetch_add(batch.transactions().size());
-      queue_.push(std::make_unique<shared_model::interface::TransactionBatch>(
-          std::move(batch)));
+      queue_.push(
+          std::make_unique<shared_model::interface::TransactionBatchImpl>(
```

Comment:
It is better to change `shared_model::interface::TransactionBatch &&` to `std::make_unique<shared_model::interface::TransactionBatch>` in method interface.